scripts/travis/maven_release.sh

```diff
@@ -16,10 +16,10 @@ git checkout -B "${TRAVIS_BRANCH}"
 git config user.email "${GIT_EMAIL}"
 
 mvn -B \
-  -Prelease,all-tas-tests,pipeline \
+  -Prelease,all-tas-tests,pipeline -Pags \
   -DreleaseVersion="${RELEASE_VERSION}" \
   -DdevelopmentVersion="${DEVELOPMENT_VERSION}" \
-  "-Darguments=-Prelease,all-tas-tests,pipeline -DskipTests -Dbuild-number=${TRAVIS_BUILD_NUMBER}" \
+  "-Darguments=-Prelease,all-tas-tests,pipeline -Pags -DskipTests -Dbuild-number=${TRAVIS_BUILD_NUMBER}" \
   release:clean release:prepare release:perform \
   -DscmCommentPrefix="[maven-release-plugin][skip ci] " \
```

Comment:
There is a separate -P here to make it more like the other projects. It also does make it simpler to find when searching.